docs/transpile/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Introduction to the transpiler
+description: Introduction to the transpiler
+
+---
+
+
+# Introduction to the transpiler
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device and optimize the circuit instructions for execution on present-day noisy quantum systems.  
+
+A central component of Qiskit, the transpiler is designed for modularity and extensibility. The goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by a **pass manager**, which schedules the passes and also allows passes to communicate with each other by providing a shared space. In this way, the transpiler opens up the door for research into aggressive optimization of quantum circuits.
+
+This documentation covers the tooling and workflows available to Qiskit users.  If you're using primitives and are only interested in st using the default transpilation options, look to the section [Configure error suppression for Qiskit Runtime.](../run/configure-runtime-compilation)
+
+
+
+## Default Transpilation
+The basic usage of the transpiler is to call the `qiskit.compiler.transpile()`](../api/qiskit/qiskit.compiler) function.  This will generate one of the present `PassManager`s based on, among other options, an `optimization_level` flag which can be set to either 0, 1, 2, or 3.  Higher levels generate more optimized circuits, at the expense of longer transpilation times.  Each of these preset `PassMangers` are composed of six basic stages:
+
+1. `init` - This pass runs any initial passes that are required before we start embedding the circuit to the backend.  This typically involves unrolling custom instructions and converting the circuit to all single- and two-qubit gates.
+1. `layout` - This pass applies a *layout*, mapping/assigning the virtual qubits in your circuit to the physical qubits of a backend.
+1. `routing` - This pass runs after a layout has been applied and will inject gates (i.e., SWAPs) in the original circuit in order to make it compatible with the backend's connectivity/coupling map.
+1. `translation` - This pass translates the gates in the circuit to the backend's basis set of instructions.
+1. `optimization` - This pass runs an optimization loop to find more efficient decompositions of your quantum circuit until a condition is met (such as a fixed depth).
+1. `scheduling` - This pass is for any hardware-aware optimizations that improve the efficiency of the executed microwave pulse instructions.
```

Comment:
Scheduling is not necessarily an optimization stage, it's only constraint is to take into account the duration of operations and ensure all the time is accounted for in the circuit. The simplest example is to just insert explicit delay instructions for any idle periods on a qubit. There's a lot of detail on what it involves here: https://qiskit.org/documentation/dev/apidoc/transpiler.html#scheduling-stage

Reply:
Ah right I see.  Is it still one of the stages that are built in the preset passmanagers?  My thought would be to include them if they are to make it clear what each stage of the preset passes consist of.  But if they don't then I don't see a need to mention it here.

Reply:
Yes, scheduling is in the preset pass managers; it's the last stage. But, by default we do a minimal amount of work in the stage, mainly to just validate we don't need to run more thorough scheduling. Most backends do scheduling better than qiskit as they have more information available to do a more complete job. But if the user sets the `scheduling_method` argument it will take care of the scheduling.

Reply:
fixed here https://github.com/Qiskit/documentation/pull/260/commits/08f47f71cd28bd014c7d86273eefa5b5e60aaf35